index.html

```diff
@@ -0,0 +1,65 @@
+<!-- Este es un comentario -->
+
+<!DOCTYPE html>
+<html lang="es">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mi CV</title>
+</head>
+
+<body>
+  <header>
+    <h1>Sarah Ines Sole Villalobos</h1>
+   </header>
+    
+    <div>
+      <p>E-mail: sisv170906@gmail.com</p>
+      <p>Tel: +57 312 768 7070</p>
+      <p>Dir: Calle 152A #54-37, Bogotá, Colombia</p>
+    </div>
+
+  <header>
+    <h2>Formación academica</h2>
+  </header>
+
+  <div> 2024/actualidad: Ingenieria informatica </div>
+  <div> Universidad de la sabana </div>  
+  <div> 2023: Bachiller </div>
+  <div> Colegio Nueva Zelandia IED </div>
+
+  <header>
+    <h2>Experiencia laboral</h2>
+   </header>
+  
+    <div> Mesera </div>
+    <div>2023/2024</div>
+    <li>Atención al cliente</li>
+    <li>Mantener el orden y la limpieza en el área de trabajo</li>
+  <li>Servicio de bebidas</li>
+       
+    <div> Atención al cliente en un local </div>
+    <div>2017/2019</div>
+    <li>Atención al cliente</li>
+    <li>Mantener el orden y la limpieza en el área de trabajo</li>
+  
```

Comment:
Al ser todos estos descendientes de una misma categoría, lo más adecuado es hacerlo una lista.

Es importante anotar que **no** puede haber un "li" fuera de un "ul" o de un "ol". "li" viene de "List Item" por lo que **siempre** debe hacer parte de una lista
```suggestion
  <section>
    <h2>Experiencia laboral</h2>
    
    <ul>
      <li>
          <p> Mesera </p>
          <p>2023/2024</p>
          <p>Atención al cliente</p>
          <p>Mantener el orden y la limpieza en el área de trabajo</p>
      </li>
    </ul>
    
    
     ...
  
     </section>
```